src/styles/style.scss

```diff
@@ -1,41 +1,45 @@
-
+// Testing
 @font-face {
-  font-family: myFirstFont;
-  src: url(../fonts/Roboto-Regular.ttf);
+  font-family: "Roboto-Regular";
+  src: url("./../fonts/Roboto-Regular.ttf") format("ttf");
 }
 
-body {
+#cExtension {
   color: #444;
   font-size: 14px;
-  font-family: myFirstFont;
+  font-family: "Roboto-Regular";
   margin: 0;
-  /* min-height: 300px; */
+  min-height: 300px;
   width: 340px;
 }
 
-h1, h2, h3, p {
-  font-family: myFirstFont;
-  margin: 0px;
-  padding: 0px;
+h1,
+h2,
+h3,
+p {
+  margin: 0;
+  padding: 0;
 }
 
 .header {
-  background-color: #0055d8;
+  // background-color: #0055D8;
+  background-color: #05668D;
```

Comment:
Given that you are using SCSS, it is good practice not to use literal colours anywhere.
It is good practice to create a `variables.scss` file and define all of your colour variables there. And then you just use the reference to the variable.

And name colours according to their function, rather than according to their value.
For example, instead of `blue` `green-light` and `grey`. Try `primary`, `secondary` and `background`.